packages/experiments/src/components/MicroFeedback/MicroFeedback.types.ts

```diff
@@ -34,6 +41,9 @@ export interface IMicroFeedbackStyles {
   //  Base styles for the root element of buttons and followup
   root?: IStyle;
 
+  //  Styles for the element containing the icons
+  iconContainer?: IStyle;
```

Comment:
Using the word "container" in styles can be a hint that this may actually be a separate component. In this case, I wonder if this should be icon slot.

Slots v2 is essentially done at this point and will be shipped with Fabric 7. I think we should consider converting this component to use it as part of the refactor. It will help clean up these style sections.